### ISSUE TITLE
iperf3 running fine, then all of a sudden starts reporting "unable to receive control message: Connection reset by peer"

### ISSUE THREAD

**Issue by katelfox** [esnet/iperf#447]
iperf3 server is running fine for a few days servicing requests, then all of a sudden starts returning "error - unable to receive control message: Connection reset by peer".  If I restart the server it will run fine again for a few days.

I'm running the following version on the server:
iperf 3.1.3
Linux ip 3.13.0-48-generic #80-Ubuntu SMP Thu Mar 12 11:16:15 UTC 2015 x86_64
Optional features available: CPU affinity setting, IPv6 flow label, TCP congestion algorithm setting, sendfile / zerocopy, socket pacing

The client is:
iperf 3.1.3
Linux ova-1 3.10.0-327.13.1.el7.x86_64 #1 SMP Thu Mar 31 16:04:38 UTC 2016 x86_64
Optional features available: CPU affinity setting, IPv6 flow label, TCP congestion algorithm setting, sendfile / zerocopy, socket pacing

Running with these options:
iperf3 -s -D -p 80 ‐‐logfile iperf.log
iperf3 --json --port 80 --client myhost

**Comment by sharpevo** [comment]
Hit it, and the logfile's null.

**Comment by katelfox** [comment]
Can you please explain your comment?  I never see anything show up in the log file.

**Comment by bmah888** [comment]
Are you still seeing this problem?  Especially with more recent code?  We've done a number of fixes to iperf3 in the meantime.

**Comment by katelfox** [comment]
A month ago or so we decided to stop using iperf3 since it was working so poorly.

**Comment by bmah888** [comment]
OK, hope you found a solution that works better for you.  Closing this issue.

**Comment by jonfcervantes** [comment]
I am seeing the same thing but it is intermittent. We are running using client (linux vm) and server (ubuntu vm) We try to run iperf3 in parallel over multiple ports. The more parallel sessions we run, the more often we see it. 
`unable to receive control message: Connection reset by peer
`
Using version iperf3 3.2
Thanks!

**Comment by bmah888** [comment]
@jonfcervantes:  Do you mind opening a new issue for the problem you're seeing?  Please include the command-lines being used (suitably sanitized if necessary).  Also could you please explain what RL and FL mean?
